tests/unit/test_vcs.py

```diff
@@ -379,12 +379,32 @@ def test_get_git_version():
     assert git_version >= parse_version('1.0.0')
 
 
+@pytest.mark.parametrize('isatty_return,use_interactive,expected', [
+    (False, None, False),
+    (False, False, False),
+    (False, True, True),
+    (True, None, True),
+    (True, False, False),
+    (True, True, True),
+])
+@patch('sys.stdin.isatty')
+def test_subversion__init_use_interactive(
+        mock_stdin, isatty_return, use_interactive, expected):
```

Comment:
I think `mock_isatty` would be better name, since that's the method you're mocking.